.github/workflows/ci.yml

```diff
@@ -14,16 +14,16 @@ jobs:
           node-version: 14
 
       - name: Install dependencies
-        run: yarn
+        run: npm install
```

Comment:
Would probably be best to use `npm ci` here

Reply:
Done! https://github.com/primer/components/pull/1250/commits/4d69bc3bd69ae5e0e9eb74a3ae091d907106e755

I also downgraded from `npm@7` → `npm@6` in that commit, per your comment that CI only supports `npm@6` (related issue: https://github.com/actions/setup-node/issues/213)